tests/Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot

```diff
@@ -421,3 +434,37 @@ Open New Notebook
     Open With JupyterLab Menu    File    New    Notebook
     Sleep    1
     Maybe Close Popup
+
+Clone Repo
+    [Documentation]    Clones git repo
```

Comment:
either this one or `Clone Git Repository` should have different names and doc, otherwise it gets confusing

Reply:
@pranavsuryaw ^